extension/chrome/elements/pgp_block_modules/pgp-block-attachmens-module.ts

```diff
@@ -91,4 +99,8 @@ export class PgpBlockViewAttachmentsModule {
     }
   };
 
+  private isAttachmentURLInvalid = (attachment: Attachment): boolean => {
+    return !!attachment.url && new URL(attachment.url).host !== 'flowcrypt.s3.amazonaws.com';
```

Comment:
There are several (two?) paths where an attachment can be initialized with `url` parameter.
One of them is `processGoogleDriveAttachments` and obviously the url `flowcrypt.s3.amazonaws.com` doesn't fit there.
The other one is `MsgBlockParser.extractFcAttachments`, so It'd be correct to exclude malicious urls in that method, by including the URL in the regex.
Please add a test with decrypting a message containing a malicious URL so we can see how it looks and  that it's not converted to an attachment.